src/lib/transport.js

```diff
@@ -37,17 +40,24 @@ function createPacketAndCondition ({
   const address = destinationAccount + '.~' + protocol + '.' + receiverId +
     (id ? ('.' + id) : '')
 
-  const blobData = omitUndefined({
-    expires_at: expiresAt,
-    data: data
-  })
+  const details = createDetails({
+    unsafeHeaders: Object.assign({
+      'Key-Id': receiverId, // TODO: is this right?
```

Comment:
Another reason to use the uuid as a nonce is to ensure that the same private headers/data won't encrypt to the same cipher-text across two different payments.